schema/generator.go

```diff
@@ -268,6 +268,18 @@ func (g *Generator) generateDDLs(desiredDDLs []DDL) ([]string, error) {
 		ddls = append(ddls, fmt.Sprintf("DROP EXTENSION %s", g.escapeSQLName(currentExtension.extension.Name)))
 	}
 
+	// Clean up obsoleted triggers
+	for _, currentTrigger := range g.currentTriggers {
+		if g.mode != GeneratorModeSQLite3 {
+			continue
+		}
```

Comment:
This PR is specific to SQLite, so I've left the status quo for databases other than SQLite. Specifically, those databases don't handle simply deleting a trigger instead of updating it. (Or is there something I'm missing?)